src/PuppeteerStringifyExtension.ts

```diff
@@ -56,7 +56,7 @@ export class PuppeteerStringifyExtension extends StringifyExtension {
     out.appendLine('');
   }
 
-  async afterAllSteps(out: LineWriter, flow: UserFlow) {
+  override async afterAllSteps(out: LineWriter, _flow: UserFlow) {
```

Comment:
does it affect the name generated in the API docs?

Reply:
Ugh, yes it does. Let me change this.